src/web/mjs/connectors/MangaParkEN.mjs

```diff
@@ -10,43 +9,121 @@ export default class MangaParkEN extends Connector {
         super.label = 'MangaPark';
         this.tags = [ 'manga', 'multi-lingual' ];
         this.url = 'https://mangapark.net';
+        this.apiURL = `${this.url}/apo/`;
+
+        this.requestOptions.headers.set('x-origin', this.url);
+        this.requestOptions.headers.set('x-referer', `${this.url}/`);
         this.requestOptions.headers.set('x-cookie', 'set=h=1;');
```

Comment:
Does this work with non URL encoded component?
`'set=h%3D1'`

Reply:
I tested with these strings (versions v3 and v5): `'set=h=1;'`,  `'set=h%3D1'` and without setting this cookie at all. In each case everything worked fine.
Maybe this cookie is no longer needed?